Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+node {
+  withCredentials([string(credentialsId: 'boundlessgeoadmin-token', variable: 'GITHUB_TOKEN'), string(credentialsId: 'sonar-jenkins-pipeline-token', variable: 'SONAR_TOKEN')]) {
+
+    currentBuild.result = "SUCCESS"
+
+    try {
+      stage('Checkout'){
+        checkout scm
+        sh """        
+          echo "Running ${env.BUILD_ID} on ${env.JENKINS_URL}"
+        """
+	sh """
+	  sh "echo ${env.android-sdk-license} > licenses/android-sdk-license"
+	  sh "echo ${env.android-sdk-preview-license} > licenses/android-sdk-preview-license"
+	"""
+      }
+
+     stage('DockerBuild'){
+           docker build -t "alpine:android-sdk" . --no-cache
+	}
+     stage('Install'){
+          docker run -it -v \$(pwd -P):/code \
+	     -w /code alpine:android-sdk sh \
+	     -c ‘bash -c “./gradlew install”’
```

Comment:
Do we want to use `uploadArchives` here instead?  I guess I'm unsure what happens after the install...are the SDK artifacts published by another Jenkins job?  I would expect them to end up [here](https://repo.boundlessgeo.com/release/com/boundlessgeo/spatialconnect) but they aren't there currently.

Reply:
No. That should be a post build for master only